packages/react-components/react-tree/src/components/TreeItem/renderTreeItem.tsx

```diff
@@ -9,13 +9,15 @@ export const renderTreeItem_unstable = (state: TreeItemState) => {
   const { slots, slotProps } = getSlots<TreeItemSlots>(state);
 
   return (
-    <slots.root {...slotProps.root}>
-      {slots.expandIcon && <slots.expandIcon {...slotProps.expandIcon} />}
-      {slots.iconBefore && <slots.iconBefore {...slotProps.iconBefore} />}
-      {slotProps.root.children}
-      {slots.iconAfter && <slots.iconAfter {...slotProps.iconAfter} />}
-      {slots.badges && <slots.badges {...slotProps.badges} />}
-      {slots.actions && <slots.actions {...slotProps.actions} />}
-    </slots.root>
+    <slots.groupper {...slotProps.groupper}>
```

Comment:
why can't the groupper attributes be added to the root slot?

Reply:
If I'm not mistaken is because the root (the treeitem) should be wrapped by the groupper attribute element